packages/zapp/console/src/components/Navigation/DefaultAppBarContent.tsx

```diff
@@ -33,37 +33,57 @@ interface DefaultAppBarProps {
 
 /** Renders the default content for the app bar, which is the logo and help links */
 export const DefaultAppBarContent = (props: DefaultAppBarProps) => {
+  const { console, items } = props;
+
   const commonStyles = useCommonStyles();
   const styles = useStyles();
 
   const isFlagEnabled = useFeatureFlag(FeatureFlag.OnlyMine);
   const { adminVersion } = useAdminVersion();
 
-  const versions: VersionInfo[] = [
-    {
-      name: t('versionConsoleUi'),
-      version: platformVersion,
-      url: `https://github.com/flyteorg/flyteconsole/releases/tag/v${platformVersion}`,
-    },
-    {
-      name: t('versionAdmin'),
-      version: adminVersion,
-      url: `https://github.com/flyteorg/flyteadmin/releases/tag/v${adminVersion}`,
-    },
-    {
-      name: t('versionGoogleAnalytics'),
-      version: t(patternKey('gaDisable', env.DISABLE_GA)),
-      url: 'https://github.com/flyteorg/flyteconsole#google-analytics',
-    },
-  ];
+  const versions: VersionInfo[] = React.useMemo(
```

Comment:
nit:
move to `import {useMemo} from 'react';` - I'm trying to get us to this way of importing standard hooks, as sometimes it saves lines 😆 not in this case, but a lot of state definitions became more readable as one-liners IMO